src/index.tsx

```diff
@@ -65,6 +67,16 @@ if (process.env.REACT_APP_MAINNET_ENV === 'production') {
   initGoogleAnalytics()
 }
 
+export const sentryRequestId = uuid()
```

Comment:
is this will be same for each session? 

btw, can u move it to constant?

Reply:
No, each session has new id.

Moved to constants.

Reply:
LOL. same for each session, I mean It only change when user refresh